SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -114,11 +116,11 @@ Plot {
 		drawGrid.vertGrid = if(gridOnY,{spec.grid},{nil});
 	}
 
-	draw {
+	draw { |mode = \linear|
```

Comment:
There are many properties that are local to each `Plot` channel. Is there a special reason that `mode` is special? Normally, I would tend to keep all the local properties as instance variable in the `Plot` object, so that we don't have to pass it around. 

It is more efficient to keep it in the object. But of course there are sometimes reasons you may want to draw the same plot object differently - but this doesn't seem the case here?

Reply:
Thanks for giving me the opportunity to rethink this.

The goal here was to add the ability to have each plot be different mode (similar to how each plot can be a different color).  When I jumped into this I tried quickly to make `plotMode` local, but there was a conflict with the way `Plot` shape-shifts when `superposed`—it becomes a single plot with multiple lines, not multiple plots overlaid. This meant that the `plotMode` would need to be passed in, or queried from `Plotter` (as it is with a single `plotMode`). I opted to be able to pass it in, assuming that there was a use case for using `Plot` without `Plotter`. This review gave me the opportunity to look closer to see that this isn't really a valid use case—`Plot` is only used through a `Plotter`—so I've modeled an update on the way `plotColor` is handled. See my comments in the main thread about how that should be submitted...